src/redo/cmd.py

```diff
@@ -12,9 +12,12 @@
 log = logging.getLogger(__name__)
 
 
-def main(argv):
+def main(argv=null):
```

Comment:
`null` is not a builtin in python, so rewrite this to `None`.

Additionally this issue was caught in CI - you can see the taskcluster UX from the PR status, in this push's case that was https://community-tc.services.mozilla.com/tasks/groups/PVev4rNPSHmAwuIgYZxwuw -- Please check that before requesting the next review.

(feel free to @ me if you get stuck in any way) -- This was a nice catch for an issue, and I'd be happy to try and help you get us to a resolution.

Reply:
I used the github web editor, sorry been writing nix

Reply:
Resolved